app/src/main/java/org/fossasia/openevent/general/search/SearchViewModel.kt

```diff
@@ -175,6 +175,7 @@ class SearchViewModel(
             }.subscribe({
                 mutableEvents.value = it
             }, {
+                mutableEvents.value = emptyList()
```

Comment:
Why?

Reply:
temporary fix for #1432

Reply:
If I don't do this, events list won't be initialized to empty favorite fragment won't get loaded.